types/mongoose/v4/index.d.ts

```diff
@@ -2660,7 +2660,7 @@ declare module "mongoose" {
       callback?: (err: any, res: T) => void): Promise<T>;
 
     /** Removes documents from the collection. */
-    remove(conditions: any, callback?: (err: any) => void): Query<void>;
+    remove(conditions: any, callback?: (err: any, result?: any) => void): Query<void>;
```

Comment:
Is it true that `remove` sometimes invokes `callback` with only 1 argument? That is the implication of `result` being optional